### DISCUSSION TITLE
Importing M3U Playlist content to Music Assistant

### DISCUSSION THREAD

**Discussion by SirRhor** [music-assistant/support#1821] [Q&A] [answered]
I have Home Assistant inside a VM on a Qnap NAS, the same for Music Assistant (obviously).
The audio files are on the NAS as well, the path being "/Public/MP3 Folder".
I have pre-made playlist on "/Public/MP3 Folder/Playlists" from MusicBee on my laptop which I can easily modify to point to the new paths.

Is there any way I can import those playlist to Music Assistant?

If not, I was thinking I can create an empty playlist on Music Assistant, modify the file itself by copying the contents of the original M3U files to it. Adding thousands of tracks to those new MASS playlist manually would take a long time and effort.
Where Music Assistant stores new playlists?

Thank you.

**Comment by OzGav** [comment, marked as answer]
Playlists are stored on your NAS. See the FAQ for playlist formats or just add a track and then look at the file to see what it looks like.
